### ISSUE TITLE
Usefulness of multimedia_access

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#124]
We currently have the required (!) property `multimedia_access`. I wonder if we really need it and its properties:

## Remote

https://github.com/tdwg/camtrap-dp/blob/9d1d85a69f84435e2ad72ef0ef9f1de4f10d5f2a/camtrap-dp-profile.json#L79-L82

Whether it is remote or local will be clear from the `file_path` in multimedia. If it starts with `http` (or `ftp`) it is remote, otherwise it is local. The same is used for Frictionless resource paths: https://specs.frictionlessdata.io/data-resource/#data-location I think we can remove this property.

## Public

https://github.com/tdwg/camtrap-dp/blob/9d1d85a69f84435e2ad72ef0ef9f1de4f10d5f2a/camtrap-dp-profile.json#L83-L86

This can differ from file to file, e.g. all images are public except for those containing humans. How should these be marked then? I would remove the property and have it be resolved by the server hosting the images.

## Auth

https://github.com/tdwg/camtrap-dp/blob/9d1d85a69f84435e2ad72ef0ef9f1de4f10d5f2a/camtrap-dp-profile.json#L87-L95

Does this provide the user with enough information to access the images? @kbubnicki any existing use cases? How would you express them using this property?

**Comment by kbubnicki** [comment]
My opinion is that there is no extra cost in providing these package-level metadata and it can be useful for quick filtering packages stored in some catalogue/repository. Otherwise these catalogues/repos would have to parse `file_path` properties to learn about where media files are stored. 

**Public** -> you are right, mixed data access policies are definitely possible but maybe this can be clarified in the project's description? I would say it applies to a majority of files, all exceptions can be mentioned in the project's description.

**Auth** -> I agree that this property is not super useful; in any case if files stored remotely are not publicly accessible (via public urls) user has to contact the data owners; can be removed?

**Comment by peterdesmet** [comment]
Given that `remote` and `auth` are not super useful, but that `public` is, maybe we should rephrase `multimedia_access` to `multimedia`, with a property `public`. It could then be extended with a property `license`:

```json
"multimedia" {
  "public": true,
  "license": [
    {
      "name": "ODC-PDDL-1.0",
      "path": "http://opendatacommons.org/licenses/pddl/",
      "title": "Open Data Commons Public Domain Dedication and License v1.0"
    }
  ]
}
```

**Comment by peterdesmet** [comment]
@kbubnicki Given that:

- `remote` and `auth` are not super useful
- Multimedia can be of mixed `public` and `private` nature (e.g. images containing humans are private)
- One can assess the public nature of the URL by testing it (e.g. does it return a 404 or not).
- It is convenient if one can change the URL from public to private and vice versa (at the image hosting server) without having to change a property in the multimedia table: e.g. a deposit with the multimedia table is published, but it is discovered that a image containing a human should be changed to private for privacy reasons.

I would:

1. Remove the whole `multimedia_access` property from the package
2. Not add a `public` property to the multimedia table.

**Comment by kbubnicki** [comment]
Ok, I think your argumentation is fair enough :) Lets drop the entire `multimedia_access` property. Maybe I think too much from the perspective of data repositories/catalogues.

**Comment by peterdesmet** [comment]
Resolved in #152
